src/main/java/cubicchunks/network/ClientHandler.java

```diff
@@ -181,12 +167,9 @@ public void handle(final PacketCubeBlockChange packet) {
 		ClientCubeCache cubeCache = worldClient.getCubeCache();
 
 		// get the cube
-		int cubeX = getX(packet.cubeAddress);
-		int cubeY = getY(packet.cubeAddress);
-		int cubeZ = getZ(packet.cubeAddress);
-		Cube cube = cubeCache.getCube(cubeX, cubeY, cubeZ);
+		Cube cube = cubeCache.getCube(packet.cubePos);
```

Comment:
I noticed that some packets seem to have cubePos as a public member, and others require access via a getter. is that intended?